jujugui/static/gui/src/app/components/entity-details/content/content.js

```diff
@@ -237,11 +237,15 @@ YUI.add('entity-content', function() {
     */
     _generateDescription: function(entityModel) {
       if (entityModel.get('entityType') === 'charm') {
+        const description = entityModel.get('description');
+        const htmlDescription = this.props.renderMarkdown(description);
```

Comment:
It fails; I'll provide a fallback value so we can gracefully handle falsey descriptions.